### DISCUSSION TITLE
Default Theme Color Problem

### DISCUSSION THREAD

**Discussion by bajrangCoder** [TomSchimansky/CustomTkinter#742] [General]
Is there any way to change the default color theme of app like the dark to light by option menu in any window

**Comment by Akascape** [comment]
Please refer this example: https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/complex_example.py

**Comment by bajrangCoder** [comment]
> 

Yes, i have done same as dark, light theme switcher for color switcher but onchage default Theme Color ,after restarting it changes...

Any solution?

**Comment by TomSchimansky** [comment]
Of course customtkinter does not save any settings after the program is closed. You have to save you settings manually, for example by writing them into a file and read this file at program start.

**Comment by bajrangCoder** [comment]
I have saved the settings into a json file and i also load it in beginning of program start. But Is there any way to update theme color within the app without quiting

**Reply by Akascape** [reply]
@bajrangCoder You can't update theme colors dynamically, but you can save that theme settings in a json file and use it in the next boot.

**Reply by bajrangCoder** [reply]
Yes , i have done something like that .

**Reply by avalon60** [reply]
I tend to use sqlite or configparser to store window geometry, theme choice etc. The saving of the geometry allows your app to remember window positions, and restore them when next started. I do this automatically when a window is closed. I also usually implement a preferences option, for things like theme, appearance mode selection,  enabling tooltips etc.
